### PR TITLE
Radiant system terminal unit classes

### PR DESCRIPTION
**Pull request by carlosduarteroa**
Added the 3 main types of radiant system terminal units.

### CODE REVIEW

**Review by gtfierro — APPROVED** [review]

**Review by gtfierro — COMMENTED** [review]
@carlosduarteroa sorry to approve this and then reconsider, but I was thinking about the use of the `_System` suffix in these classes. Is there any analogous component to "terminal unit" in ESS and TABS installations? A "segment" or "loop'" or something similar? I feel that might be a better fit for an Equipment class, but I'm open to ideas

**Comment by carlosduarteroa** [comment]
I am not sure if you saw an earlier code line comment I made on this similar issue that you are raising on the use of `_System`. But we could replace the suffix to `_Panel` to distinguish it as the terminal unit as opposed to a collection of panels. I agree, the intention is for this class to be an equipment entity.

File: bricksrc/equipment.py [hunk 1]
```diff
@@ -249,6 +249,41 @@
                 },
             },
             "VAV": {"tags": [TAG.Equipment, TAG.VAV]},
+            "Radiant_Panel": {
+                "tags": [TAG.Equipment, TAG.Radiant, TAG.Panel],
+                "subclasses": {
+                    "Thermally_Activated_Building_System": {
```
[the remark this answers is not in the document]
Comment (gtfierro) [reply bricksrc/equipment.py:255]: Yes, for sure! Can we do the `_Panel` suffix?

[the remark this answers is not in the document]
Comment (carlosduarteroa) [reply bricksrc/equipment.py:255]: Yes, I like the `_Panel` suffix. I will make the changes.

**Review by gtfierro — APPROVED** [review]

**Pull request merged**